src/host/ft_host/InitTests.cpp

```diff
@@ -280,7 +281,7 @@ MODULE_SETUP(ModuleSetup)
         }
     };
 
-    VERIFY_IS_LESS_THAN(tries, 100, L"Make sure we set up the new console in time");
+    VERIFY_IS_LESS_THAN(delay, 30000u, L"Make sure we set up the new console in time");

```

Comment:
Another option would be to just `return true` above instead of `break`ing. That way you can blindly `VERIFY_FAIL()` here.